mne_bids/dig.py

```diff
@@ -25,7 +25,7 @@
 
 
 def _handle_electrodes_reading(electrodes_fname, coord_frame,
-                               coord_unit, raw, verbose):
+                               coord_unit, verbose):
     """Read associated electrodes.tsv and populate raw.
```

Comment:
you are not populating "raw" anymore, right?

Reply:
Yes, I think since `_read_dig_bids` is the parent function to coordinate the `*coordsystem.json` and `*electrodes.tsv`, then the setting of raw should occur outside.